rabbitmq/configure.sls

```diff
@@ -0,0 +1,128 @@
+#!pyobjects
+from __future__ import division
+
+import re
+import socket
+import collections
+from bisect import bisect
+
+include('rabbitmq.service')
+
+
+def update(d, u):
+    for k, v in u.items():
+        if isinstance(v, collections.Mapping):
+            r = update(d.get(k, {}), v)
+            d[k] = r
+        elif isinstance(v, list):
+            d[k] = d.get(k, [])
+            d[k] += v
+        else:
+            d[k] = u[k]
+    return d
+
+
+def is_ipaddr(addr_string):
+    try:
+        socket.inet_pton(socket.AF_INET, addr_string)
+    except (AttributeError, OSError, socket.error):
+        try:
+            socket.inet_pton(socket.AF_INET6, addr_string)
+        except (AttributeError, OSError, socket.error):
+            return False
+    return True
```

Comment:
Yeah, I had to google around for that one since the ipaddress module isn't present in 2.7.